projects/storefrontlib/src/cms-components/product/product-tabs/product-reviews/product-reviews.component.ts

```diff
@@ -37,11 +46,19 @@ export class ProductReviewsComponent {
 
   initiateWriteReview(): void {
     this.isWritingReview = true;
+
+    setTimeout(() => {
```

Comment:
Can we do this another way without `setTimeout`? Its use is a big nono for SSR.
https://sap.github.io/cloud-commerce-spartacus-storefront-docs/server-side-rendering-coding-guidelines/#working-with-timeouts